mypy/checker.py

```diff
@@ -3923,11 +3923,11 @@ def find_isinstance_check_helper(self, node: Expression) -> Tuple[TypeMap, TypeM
             return {}, None
         elif is_false_literal(node):
             return None, {}
-        elif isinstance(node, CallExpr):
+        elif isinstance(node, CallExpr) and len(node.args) > 0:
```

Comment:
Maybe instead add a separate 'if' statement under here that does `if len(node.args) == 0: return {}, {}`? 

So that way, we don't have to do a bunch of extra `isinstance(...)` checks if the node happens to be an unrelated CallExpr.